### PR TITLE
[sysutils/lsof] major update

### PR DESCRIPTION
**Pull request by lrosenman**
Status report for sysutils/lsof

### CODE REVIEW

File: lsof.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+=== sysutils/lsof major upgrade
+
+Links
+link:https://github.com/lsof-org/lsof[lsof project repo] URL: link:https://github.com/lsof-org/lsof[https://github.com/lsof-org/lsof] +
+
+Contact: Larry Rosenman <ler@FreeBSD.org>
+
+sysutils/lsof in ports has had a major upgrade to no longer go looking 
+in /dev/kmem for data and using userspace API's.  This has taken a long
```
Comment (pauamma) [inline lsof.adoc:9]:
```suggestion
in /dev/kmem for data and using the userland API.  This has taken a long
```
Single API, comprising a bunch of system calls, device data formats, etc. Or if you see it as a file IO API, a process control API, a resource allocation API, etc., that is one API per functional group, I'd still remove the ': APIs.

File: lsof.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+time to hit the tree, but is finally done.  This fixes lsof(1) to work with
+ZFS again for the first time since 13.0-RELEASE.
+
+This will make the maintenance of sysutils/lsof much easier going forward.
+
+To the kernel folks: if you make changes that break lsof, please submit a 
+github PR to https://github.com/lsof-org/lsof.
```
Comment (pauamma) [inline lsof.adoc:16]: If there's a way to find out about changes breaking lsof, best to mention it here, I think.

File: lsof.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+My thanks to Warner Losh <imp@FreeBSD.org>, Mateusz Guzik <mjg@FreeBSD.org>,
+and Ed Maste <emaste@FreeBSD.org> for help getting this major change landed. 
+
+
+
```
Comment (pauamma) [inline lsof.adoc:22]: Spurious extra linebreaks.

**Review by pauamma — CHANGES_REQUESTED** [review]

**Comment by lrosenman** [comment]
I believe I have resolved all the comments

**Review by pauamma — APPROVED** [review]
Yep, LGTM now.

**Pull request merged**